twingate/README.md

```diff
@@ -36,7 +36,14 @@ This integration allows organizations to monitor a user's resource access activi
 4. Restart the Datadog Agent by running `service datadog-agent restart`.
 5. Confirm that the Twingate Analytic log appears in the [Log Explorer][10].
 
+### Setup Datadog Log Pipeline
+1. Navigating to **Logs** > **Pipeline**
+2. Search for Twingate
+3. Clone the pipeline
```

Comment:
These aren't the steps for setting up a pipeline, but rather for creating a new pipeline to customize. Would you be open to expanding upon the sorts of customizations that users might want to make in this case?

Reply:
Hi @buraizu, thank you for your feedback.

the pipeline is already part of the Datadog pipeline library and no additional customisation needed.

Reply:
Thanks for confirming that. As such, the pipeline should be automatically enabled when logs arrive with the designated `source` attribute, as mentioned in the documentation here:
https://docs.datadoghq.com/logs/log_configuration/pipelines/?tab=source#integration-pipelines
The steps provided would create a clone of the existing pipeline in order to make modifications. Since you're not suggesting any modifications to the existing pipeline, this section can be removed.